yw_website/yw_website/apps/website/templates/includeable/workflow_list.html

```diff
@@ -31,6 +31,12 @@
                 </div>
                 <div class = "title">
                     {{ workflow.title|linebreaks }}
+                    {% if edit == True %}
+                        <form method="get" action=" {% url 'edit_page' workflow.id workflow.version_id %}" style="align-self: right; margin-right: 2%; border-radius: 10px;">
```

Comment:
I'd like to see the style used in this line in the edit page css instead, nice use of if edit

Reply:
there is still is still styling in the html here, we'd like to see it in a css file